packages/stress/src/utils/stressClient.ts

```diff
@@ -80,7 +82,7 @@ export class StressClient {
         public globalPersistedStore: IStorage | undefined,
         public storageKey: string,
     ) {
-        logger.log('StressClient', {
+        logger.info('StressClient', {
             clientIndex,
```

Comment:
@miguel-nascimento we should ideally have all logs coming from the same client get annotated by the same clientId. i think we can set a class property on `StressClient.logger = getLogger('stress:stressClient', { clientId: this.clientId }` and have all class methods use `this.logger`